api-tests/secure-pipeline/run-tests.sh

```diff
@@ -19,6 +19,12 @@ export CRI_STUB_GEN_CRED_API_KEY
 MANAGEMENT_TICF_API_KEY=$(aws secretsmanager get-secret-value --secret-id /build/core/credentialIssuers/ticf/connections/stub/apiKey | jq -r .SecretString)
 export MANAGEMENT_TICF_API_KEY
 
+MANAGEMENT_TICF_API_KEY=$(aws secretsmanager get-secret-value --secret-id /build/core/credentialIssuers/ticf/connections/stub/apiKey | jq -r .SecretString)
+export MANAGEMENT_TICF_API_KEY
+
+MANAGEMENT_CIMIT_STUB_API_KEY=$(aws ssm get-parameter --name /tests/core-back-build/cimit_api_key | jq -r .Parameter.Value)
```

Comment:
Do we expect these `/tests/core-back-build/cimit_api_key` (see [here](https://github.com/govuk-one-login/ipv-core-common-infra/blob/7b04ca09bbe46a229aa4c045999b950f77f99690/utils/config-mgmt/app/configs/core.build.params.yaml#L117)) to be removed once the api tests have enough coverage? In which case, it might be better to add this to secret manager instead?